dev/profile/style.scss

```diff
@@ -0,0 +1,100 @@
+.profile-wrap {
+  width: 100%;
+  min-height: 100%;
+
+  .title-container {
+    text-align: center;
+  }
+
+  .profile-container {
+    position: relative;
+  }
+
+  .button-container { 
+    text-align: center;
+  }
+
+  .profile-photo,
+  .company-logo {
+    border: 1px solid rgba(0, 0, 0, 0.26);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .profile-photo {
+    width: 320px;
+    height: 320px;
+    margin: 0 auto 20px auto; 
+    background-size: cover;
+    background-image: url('/files/images/account_photo.svg');
+  }
+
+  .upload-photo {
+    position: absolute;
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    outline: none;
+    border: 2px solid rgba(0, 0, 0, 0.26);
+    top: 260px;
+    right: 50%;
+    margin-right: -25px;
+    text-align: center;
+    line-height: 50px;
+    background-color: #fff;
+
+    i {
+      vertical-align: middle;
+    }
+  }
+
+  .second-name {
+    margin-left: 56px;
+  }
+
+  .errorMessage {
+    text-align: center;
+  }
+
+  .personal-info,
+  .company-info {
+    padding: 0 25px;
+  }
+
+  .personal-info {
+    border-bottom: 1px solid rgba(0, 0, 0, 0.26);
+  }
+
+  .company-logo {
+    display: inline-block;
+    width: 200px;
+    height: 50px;
+    margin-right: 10px;
+    background-image: url('/files/images/company_logo.svg');
+    background-size: contain;
+  }
+}
+
+@media (min-width: 600px) {
```

Comment:
Please group classes in one place 
.className {
  prop: value1;
  @media (min-width: 600px) {
    prop: value2;
  }
}